scripts/apache/get_fasta_sequence

```diff
@@ -66,6 +67,11 @@ sub get_fasta_sequence {
         if ($i == 0) {
             $cut_id =~ s/\.\d+//;
         }
+        
+        if($seq_target eq 'dna' && index($queries[$i], 'uniprotkb') != -1 ) {
+            $i++;
+            next;
+        }
```

Comment:
Please rename everywhere seq_target to seq_type, dna or protein - is a type. 
And also this particular piece of coede depends on string constant mentioned above and can be changed. 
Just when we push queries to the list - push only needed, don't check on every iteration here if we should keep it or skip.